conans/test/functional/toolchains/cmake/test_file_api.py

```diff
@@ -8,6 +8,7 @@
 from conans.test.utils.tools import TestClient
 
 
+@pytest.mark.xfail(reason="NewCppInfo: xfail until it adds default folders for the components")
```

Comment:
Yes, the CMakeFileAPI implementation fails because we removed the defaults in folder for components, this is something I imagine will be easy to fix later and @SSE4 knows that code I did not wanted to get into it because I have not so much experience with that part.